ts/RootContainer.tsx

```diff
@@ -38,13 +38,13 @@ class RootContainer extends React.PureComponent<Props> {
 
   public componentDidMount() {
     if (Platform.OS === "android") {
+      // If the application is summoned from a deep link intent.
       Linking.getInitialURL()
         .then(this.navigateToUrlHandler)
         .catch(console.error); // tslint:disable-line:no-console
-    } else {
-      Linking.addEventListener("url", this.handleOpenUrlEvent);
     }
 
+    Linking.addEventListener("url", this.handleOpenUrlEvent);
```

Comment:
we should probably remove the event listener on `componentWillUnmount`

Reply:
actually, this gets executed also for Android, do we need an `else` ?

Reply:
This is the part that fixes the changed behaviour introduced by https://github.com/teamdigitale/italia-app/pull/338. Now, this is the listener for both iOS and Android that handles the deep link navigation event.